com.unity.perception/Runtime/GroundTruth/Labelers/KeyPointLabeler.cs

```diff
@@ -79,14 +79,14 @@ public KeyPointLabeler(IdLabelConfig config, KeyPointTemplate template)
         protected override void Setup()
         {
             if (idLabelConfig == null)
-                throw new InvalidOperationException("KeyPointLabeler's idLabelConfig field must be assigned");
+                throw new InvalidOperationException($"{GetType().Name}'s idLabelConfig field must be assigned");
```

Comment:
Why this change? `nameof(KeypointLabeler)` is more appropriate if you don't expect inheritance.

Reply:
Good point, I'll change that.